scala/lambdas/ingest-validate-generic-ingest-inputs/src/test/scala/uk/gov/nationalarchives/ingestvalidategenericingestinputs/MetadataJsonValueValidatorTest.scala

```diff
@@ -44,6 +44,9 @@ class MetadataJsonValueValidatorTest extends AnyFlatSpec with MockitoSugar with
     ("Files", Map(3 -> 0, 4 -> 0), "ArchiveFolder")
   )
 
+  private val casesWhereEntryShouldBeLabelledAsMetadataFile: TableFor2[Boolean, String] =
+    Table(("metadataFile value is true", "id is in 'originalMetadataFiles' array"), (true, "is"), (true, "is not"), (false, "is"))
```

Comment:
Can you not use `true` and `false` instead of `is` and `is not` I know it slightly messes up your test name but it makes the code a bit easier to parse?